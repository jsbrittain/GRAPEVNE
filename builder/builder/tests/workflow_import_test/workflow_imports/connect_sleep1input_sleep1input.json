[
  {
    "id": "idcode",
    "name": "sleep1input_test1",
    "type": "module",
    "config": {
      "snakefile": "builder/tests/workflow_import_test/modules/sleep1input/workflow/Snakefile",
      "params": {
        "input_filename": "mark",
        "input_namespace": "in",
        "output_filename": "mark",
        "output_namespace": "out",
        "sleeptime": 0
      }
    }
  },
  {
    "id": "idcode",
    "name": "sleep1input_test2",
    "type": "module",
    "config": {
      "snakefile": "builder/tests/workflow_import_test/modules/sleep1input/workflow/Snakefile",
      "params": {
        "input_filename": "mark",
        "input_namespace": "in",
        "output_filename": "mark",
        "output_namespace": "out",
        "sleeptime": 0
      }
    }
  },
  {
    "name": "Join [Sleep1input-to-Sleep1input]",
    "type": "connector",
    "config": {
      "map": ["sleep1input_test1", "sleep1input_test2"]
    }
  }
]
