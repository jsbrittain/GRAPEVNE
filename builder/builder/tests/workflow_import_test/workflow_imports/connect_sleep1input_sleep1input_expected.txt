config={}
config["input_namespace"]={}
config["input_namespace"]["$sleep1input_test1"]="in"
config["output_namespace"]="sleep1input_test2"
config["sleep1input_test1"]={}
config["sleep1input_test1"]["config"]={}
config["sleep1input_test1"]["config"]["input_filename"]="mark"
config["sleep1input_test1"]["config"]["input_namespace"]="in"
config["sleep1input_test1"]["config"]["output_filename"]="mark"
config["sleep1input_test1"]["config"]["output_namespace"]="sleep1input_test1"
config["sleep1input_test1"]["config"]["sleeptime"]="None"
config["sleep1input_test1"]["name"]="sleep1input_test1"
config["sleep1input_test1"]["snakefile"]="builder/tests/workflow_import_test/modules/sleep1input/workflow/Snakefile"
config["sleep1input_test1"]["type"]="module"
config["sleep1input_test2"]={}
config["sleep1input_test2"]["config"]={}
config["sleep1input_test2"]["config"]["input_filename"]="mark"
config["sleep1input_test2"]["config"]["input_namespace"]="sleep1input_test1"
config["sleep1input_test2"]["config"]["output_filename"]="mark"
config["sleep1input_test2"]["config"]["output_namespace"]="sleep1input_test2"
config["sleep1input_test2"]["config"]["sleeptime"]="None"
config["sleep1input_test2"]["name"]="sleep1input_test2"
config["sleep1input_test2"]["snakefile"]="builder/tests/workflow_import_test/modules/sleep1input/workflow/Snakefile"
config["sleep1input_test2"]["type"]="module"

module sleep1input_test1:
    snakefile:
        config["sleep1input_test1"]["snakefile"]
    config:
        config["sleep1input_test1"]["config"]
use rule * from sleep1input_test1 as sleep1input_test1_*

module sleep1input_test2:
    snakefile:
        config["sleep1input_test2"]["snakefile"]
    config:
        config["sleep1input_test2"]["config"]
use rule * from sleep1input_test2 as sleep1input_test2_*
